ehsm-service/server.js

```diff
@@ -0,0 +1,182 @@
+const express = require('express')
+const ffi = require('ffi-napi');
```

Comment:
change the file name server.js -> ehsm_kms_server.js

Reply:
It will be fix.